### PR TITLE
docs: Update the README

### PR DESCRIPTION
**Pull request by ruthmoog**
Big update to the readme. Needs updates re instructions for Ruby Critic @rhysco8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +15,65 @@ First, clone this repository. Then:
...
+-----------
+
+- Anna Sobolewska
+- Ben Scott
+- Chris Groves
+- Rhys Collier
+- Ruth Baker
+- Syed A. Shah
+
+## Liscence
```
Comment (rhysco8) [inline README.md]:
```suggestion
## Licence
```

File: README.md [hunk 2]
```diff
@@ -24,7 +15,65 @@ First, clone this repository. Then:
...
+## Running the tests
+
+### End to end tests
+
+- Run `rspec` in the command line to see test passing and test coverage.
+
+### Code styling tests
+
+- Run `rubocop` in the command line to see any styling offences.
+
```
Comment (rhysco8) [inline README.md:29]: Here's the instructions for rubycritic
```suggestion
### Code quality tests

- Run `rubycritic` in the command line to see any code quality issues such as smells and high complexity lines. This report will open in the browser. To alternatively see the results in the console, run `rubycritic - f console` instead
```

**Review by rhysco8 — CHANGES_REQUESTED** [review]
Looks great, thanks for updating this Ruth. I have suggested 2 changes, including adding in the rubycritic section

**Review by bengscott2 — APPROVED** [review]
Way to go both of you! Dream Team.

**Review by chris-groves — APPROVED** [review]

**Review by rhysco8 — APPROVED** [review]
Nice

**Pull request merged**